OneSchedule.Domain.Abstractions/INotificationScheduler.cs

```diff
@@ -0,0 +1,14 @@
+﻿using OneSchedule.Entities;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace OneSchedule.Domain.Abstractions
+{
+    public interface INotificationScheduler
+    {
+        public Task ScheduleNotifications(EventEntity eventEntity);
```

Comment:
You don't need to specify access modifiers in interface. They are public by default